.github/workflows/ci.yaml

```diff
@@ -49,3 +49,42 @@ jobs:
 
       - name: Test
         run: pnpm test
+
+  continious-deployment:
+      if: github.event_name == 'workflow_dispatch'
+
+      name: 'Continous Deployment'
+      runs-on: ubuntu-latest
+      needs: ['continous-integration']
+
+      steps:
+          - uses: actions/checkout@v4
+
+          - uses: pnpm/action-setup@v4
+            with:
+                version: 9
+
+          - name: Install dependencies
+            run: pnpm install
+
+          - name: Build
+            run: pnpm build
+
+          - name: Set git configuration 
+            run: |
+              git config user.name "Animo Solutions"
+              git config user.email "development@animo.id"
+
+          - name: Publish
+            id: publish
+            env:
+              NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            run: |
+              npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN
+
+              if [ ${{ inputs.release-type }} == 'alpha' ]; then
+                pnpm release --preRelease=alpha
```

Comment:
do we also need to provide the release type here? E.g. if we're on 1.0.0 now, if we do alpha release should it be 1.0.1-alpha.0, 1.1.0-alpha.0, or 2.0.0-alpha.0?